batch/batch/worker/worker.py

```diff
@@ -1964,13 +1982,12 @@ async def run(self):
             # file with explicit versioning to make sure we maintain backwards compatibility.
             self.write_batch_config()
 
+            mjs_fut = self.mark_started()
```

Comment:
hmm. I wonder why we didn't do this before the JVM acquisition before. Seems better this way.